packages/core/src/components/progress/progress.story.tsx

```diff
@@ -0,0 +1,33 @@
+import { c, classy, m, ProgressProps as BaseProps } from '@onfido/castor';
+import { html } from '../../../../../docs';
+
+export interface ProgressProps extends BaseProps {
+  children?: string | null;
+  'aria-valuetext'?: string;
+}
+
+export const Progress = ({
+  size,
+  value,
+  max = 100,
+  hideLabel,
+  children,
+  'aria-valuetext': ariaValuetext,
+  ...props
+}: ProgressProps) =>
+  html('div', {
+    ...props,
+    class: classy(c('progress'), m(size)),
+    role: 'progressbar',
+    'aria-valuenow': String(value),
+    'aria-valuemin': String(0),
+    'aria-valuemax': String(max),
+    'aria-valuetext':
+      ariaValuetext || (typeof children === 'string' ? children : undefined),
+    style: `--percent-value: ${Math.round((value / max) * 100)}%`,
```

Comment:
Maybe new utility could be used?

Also, I believe calculation is not up to date with the React version here.

Reply:
Nice catch! Yes! An utility would be nice in both `core` where we have to set the values manually, and `react` when we want to set a custom label but also show the percentage.

I'm thinking in something like `toProgressPercentValue(min, max, value): string`, what do you think?

Also, not sure where to put it 🤔

Reply:
Sorry, I just meant the `cssVars` utility...

Reply:
Ah sorry! Lost the context a bit in the morning 🙈 

About using the `cssVars` here, different from the `react` code, the `html` helper is only to generate a helper right?

If this is the case, I don't see an advantage of using here, for two reason:
 
 - The cssVars generate react objects, like `{ percentageValue: '40%' }`, we would have to map it to strings
 - Simply setting the string may be more readable than going through function descriptions in order to understand what's happening
 
 So I think it's doable, but it would require more code here or another utility. What do you think?

Reply:
That's fine, don't worry.

I guess it's only a matter of aligning with React so it's using both `min` and `max`.

Reply:
The calculation is fixed here and in another line after that, thanks!